contract/models/contract.py

```diff
@@ -55,8 +55,10 @@ def _compute_price_subtotal(self):
     @api.one
     @api.constrains('discount')
     def _check_discount(self):
-        if self.discount > 100:
-            raise ValidationError(_("Discount should be less or equal to 100"))
+        for line in self:
```

Comment:
Pylint error in Travis:

************* Module contract.models.contract

contract/models/contract.py:57: [W8104(api-one-deprecated), AccountAnalyticInvoiceLine._check_discount] api.one deprecated

pylint expected errors 0, found 1!